### PR TITLE
Monitoring

### PR DESCRIPTION
**Pull request by pbrisbin**
- Fixes a problem with logging
- Adds StatsD metrics
  - request
  - request.<topic>
  - processing_time
  - success or error

### CODE REVIEW

File: src/main/java/nc/isi/http2kafka/Server.java [hunk 1]
```diff
@@ -108,26 +118,37 @@ public Server(Properties producerProperties) {
...
 			ensurePost(req);
 
 			String topic = extractTopic(req);
 			String key = extractKey(req);
 
+			statsd.incrementCounter("request." + topic);
 			sendBody(req, topic, key);
 
+			long endTime = System.nanoTime();
+			long duration = (endTime - startTime) / 1000000; // ms
```
Comment (gdiggs) [inline src/main/java/nc/isi/http2kafka/Server.java:139]: Could you use [this function](http://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis%28%29) instead?

Comment (pbrisbin) [inline src/main/java/nc/isi/http2kafka/Server.java:139]: Probably, but [this](http://stackoverflow.com/questions/351565/system-currenttimemillis-vs-system-nanotime) has reasons.

Comment (gdiggs) [inline src/main/java/nc/isi/http2kafka/Server.java:139]: Ah interesting difference between them

**Comment by gdiggs** [comment]
LGTM

**Pull request merged**